web/src/pages/apd/security-planning/SecurityPlanningReadOnly.test.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { renderWithConnection, screen } from 'apd-testing-library';
+import { mockFlags, resetLDMocks } from 'jest-launchdarkly-mock';
+
+import { APD_TYPE } from '@cms-eapd/common';
+
+import SecurityPlanningSummary from './SecurityPlanningReadOnly';
+
+const defaultProps = {
```

Comment:
maybe name this emptyState or something because we use the name defaultProps everywhere else to talk about props being sent into the container